ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__multi-nightly.yaml

```diff
@@ -1427,13 +1427,19 @@ tests:
   cron: 8 8 29 2 *
   steps:
     cluster_profile: gcp-qe-c3-metal
+    env:
+      E2E_RUN_TAGS: not @storage
+      TEST_FILTERS: ~STORAGE&
```

Comment:
I think `E2E_RUN_TAGS` amend the filter here to the default one in `cucushift-e2e`, so it's fine.
While `TEST_FILTERS`  override the default one in `openshift-extended-tests`, which will include longduration and distruptive test cases. I think that might be the reason why 8+ hours passed, only 100+ test cases executed, while 1000+ test cases are still pending.
IMO, the `TEST_FILTERS` should be set to `xxx-long-list-of-default-values-xxx;~STORAGE&`

CC @kuiwang02 in case what I said is wrong here.

Reply:
@liangxia is right.

cc @Phaow @memodi @geliu2016 @Xia-Zhao-rh to be aware of this because I am not working on it any more.

Reply:
@liangxia @kuiwang02 Thanks a lot.

Reply:
@kuiwang02 Acked and thanks!